onnxruntime/python/onnxruntime_pybind_ortvalue.cc

```diff
@@ -341,6 +341,7 @@ void addOrtValueMethods(pybind11::module& m) {
             v->push_back(FromDlpack(dlpack_tensor.ptr(), is_bool_tensor));
           },
           "Add a new OrtValue after being ownership was transferred from the DLPack structure.", py::arg("dlpack_tensor"), py::arg("is_bool_tensor") = false)
+#endif
       .def(
           "push_back_batch", [](std::vector<OrtValue>* v, std::vector<py::object>& torch_tensors, std::vector<int64_t>& data_ptrs, std::vector<py::object>& element_types, const std::vector<std::vector<int64_t>>& shapes, const std::vector<OrtDevice>& devices) {
```

Comment:
are there any relevant tests which should also be moved outside of training builds?